modules/apps/questions/questions-web/test/js/components/Answer.es.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+import Answer from '../../../src/main/resources/META-INF/resources/js/components/Answer.es';
+
+import '@testing-library/jest-dom/extend-expect';
+import {waitForElementToBeRemoved} from '@testing-library/dom';
+
+import {markAsAnswerMessageBoardMessageQuery} from '../../../src/main/resources/META-INF/resources/js/utils/client.es';
+import {renderComponent} from '../../helpers.es';
+
+const mockAnswer = {
+	actions: {
+		delete: {
+			operation: 'deleteMessageBoardMessage',
```

Comment:
do you need to mock this?

Reply:
I mocked this because I need to provide an Answer to the component.